### ISSUE TITLE
Blue field phenomenon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10589]
**Preferred term label:**

**Synonyms**
Blue field entoptic phenomenon
Scheerer phenomenom

**Definition (free text, please give PubMed ID)**
PMID: 34671311 The so-called “**blue field phenomenon**” is characterized by a perception of moving fragments against a bright
background, such as the sky. It is the autovisualization of the own leukocytes in the retinal capillaries (26). It has been shown that about 42% (27) to 79% (4) of VSS patients experience this symptom, also in an excessive magnitude.

PMID: 29140814 Enhanced entoptic phenomena. At least one of the following: excessive floaters in both eyes, excessive blue field entoptic phenomenon, self-light of the eye, or spontaneous photopsia. **Entoptic phenomena** arise from the structure of the visual system itself. **The blue field entoptic phenomenon** is described as uncountable little grey/white/black dots or rings shooting over visual field in both eyes when looking at homogeneous bright surfaces, such as the blue sky; self-light of the eye is described as
colored waves or clouds when closing the eyes in the dark; spontaneous photopsia is characterized by bright flashes of light.

**Parent term (use hpo.jax.org/app)**
 Entoptic phenomenon (see issue #10588 )

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:420556 Visual snow syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
adding as
HP:0020331
Blue-field entoptic phenomenon
Def: Unremitting, positive visual phenomena, present in the entire visual field and characterized by uncountable tiny flickering dots interposed between the person's vision and the background. This ‘static’ is typically black and white but can also be colored, flashing, or transparent. In addition to the static, or snow, patients can experience additional visual symptoms of either direct neurological origin, such as palinopsia, photophobia, and nyctalopia (i.e., impaired night vision), or that arise from the optic apparatus.
PMID: 29140814
